site/zenodo_rdm/legacy/serializers/__init__.py

```diff
@@ -33,6 +33,12 @@ def unwrap_hits(self, data, many, **kwargs):
         return data.get("hits", {}).get("hits", [])
 
 
+class ZenodoListSchema(BaseListSchema):
```

Comment:
nit: Seeing the class below prefixed `Legacy`, and this one not prefixed, made me think that `ZenodoListSchema` was related to the modern API (until I realized that the package name made all this code legacy).
Not needed to be changed since other existing classes like `ZenodoSchema` are not prefixed neither.

Reply:
Agree that the naming is not consistent throughout the legacy module (because of a code move/rename), and there's also some special naming/meaning around the terms "Legacy" and "ZenodoJSON" (the latter is the case for this class).

There's a bit of a counter-incentive also on putting effort into polishing the legacy layer since it's "throwaway" code that will be pruned once we deprecate the legacy REST API.